src/Rules/ValueUniqueOther.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace atk4\validate\Rules;
+
+use atk4\data\Exception;
+use atk4\data\Model;
+
+/**
+ * Class ValueUniqueOther.
+ *
+ * Validate : if value is unique in Model in another field
+ *
+ * Rule code : atk4_value_unique_other
+ *
+ * @example ['atk4_value_unique_other',$model,'field_name_to_check']
+ */
+class ValueUniqueOther extends ValueUnique
+{
+    public static $rule_id = 'atk4_value_unique_other';
+
+    public static function getCallback($field, $value, array $params, array $fields): bool
+    {
+        if (count($params) != 2) {
+            throw new Exception(
+                [
+                    'Rule '.self::$rule_id.' must have 2 parameters ( Model and Model field to check against)',
+                    'field'  => $field,
+                    'value'  => $value,
+                    'params' => $params,
+                    'fields' => $fields,
+                ]
+            );
+        }
+
+        $ref_model = $params[0] ?? null;
+        $ref_model_field = $params[1] ?? null;
+
+        if (!($ref_model instanceof Model)) {
+            throw new Exception(
+                [
+                    'Rule '.self::$rule_id.' first param must be a atk4\data\Model (the Model to check against)',
+                    'field'  => $field,
+                    'value'  => $value,
+                    'params' => $params,
+                    'fields' => $fields,
+                ]
+            );
+        }
+
+        if (!is_string($ref_model_field)) {
+            throw new Exception(
+                [
+                    'Rule '.self::$rule_id.' second param must be string (the Model field to check)',
+                    'field'  => $field,
+                    'value'  => $value,
+                    'params' => $params,
+                    'fields' => $fields,
+                ]
+            );
+        }
+
+        $isLoaded = $ref_model->loaded(); // if loaded ? INSERT : UPDATE
+
+        $count = (int) $ref_model->newInstance()
+                                ->addCondition($ref_model_field, '=', $value)
```

Comment:
can you provide a real-life example (simple one), if you can't come up with a good one - then lets exclude the rule (as it wouldn't be helpful)